bus-mapping/src/circuit_input_builder.rs

```diff
@@ -421,7 +428,7 @@ impl CircuitInputBuilder<FixedCParams> {
         if is_first_chunk {
             push_op(
                 &mut state.block.container,
-                &mut end_block_last,
+                &mut end_block,
```

Comment:
why don't we just move `StartOp` and `PaddingOp` here in to the padding `ExecStep`?